dask/bag/text.py

```diff
@@ -33,6 +34,9 @@ def read_text(path, blocksize=None, compression='infer',
     linedelimiter: string
     collection: bool, optional
         Return dask.bag if True, or list of delayed values if false
+    storage_options: dict
+        Extra options that make sense to a particular storage connection, e.g.
+        host, port, username, password, etc.
     **kwargs: dict
         Extra parameters to hand to backend storage system.
         Often used for authentication when using remote storage like S3 or HDFS
```

Comment:
This actually revealed a bug in tests: https://github.com/dask/dask/blob/master/dask/bag/tests/test_bag.py#L478

There is no `lineterminator`, but rather `linedelimiter`. It turned out that API is inconsistent here:

* `lineterminator` in `dask.dataframe.read_csv`
* `linedelimiter` in `dask.bag.read_text`
* `delimiter` in `dask.bytes.read_bytes`

For this PR I will just fix the test, but it should be addressed.